back/app/models/area.rb

```diff
@@ -85,7 +86,20 @@ def sanitize_description_multiloc
     self.description_multiloc = service.linkify_multiloc description_multiloc
   end
 
+  def sanitize_title_multiloc
+    return unless title_multiloc&.any?
```

Comment:
I should be ok to sanitize an empty hash or a hash with empty values.
```suggestion
    return unless title_multiloc.nil?
```

Reply:
Ufa, how did I miss that! 🤦